readme.md

```diff
@@ -1,31 +1,29 @@
 # Pokenatra
 
-JANcorp is in dire need of your help. Turns out, wamazon wasn't entirely successful. We decided that it would be better to ride on the coat tails of the lastest craze, Pokemon Go. We want YOU to create a web app that mirrors much of the same functionality as the popular mobile app!
-
 ## Objectives
 
-Build at least a single model application on the Sinatra Application that allows for the creating, reading, updating and deleting of a Pokemon.
+Build at least a single model application using Sinatra that allows for the creating, reading, updating and deleting of a Pokemon.
 
-Listed below are the various potential ERD's for this app in increasing difficulty. Do NOT attempt to code an ERD UNLESS you have already finished coding the domains before it. IE. I can only do silver if I've finished the MVP. I can only do gold if I've finished silver. Remember, a polished MVP is better than a non functioning silver. It should be also noted that
+Listed below are the various potential ERD's for this app in increasing difficulty. **Do not attempt to code an ERD unless you have already finished coding the domains before it** (i.e., "I can only do silver if I've finished the MVP"). Remember, a polished MVP is better than a non functioning silver.
 
 ![pokenatra_erd](pokenatra_erd.png)
 
 > WARNING: type is a reserved word in Active Record, make sure that you never a column name type in Active Record.
 
 ## Deliverables
-- Fully functional CRUD on a single model(Pokemon)
 
-## Steps to get started
-Step 1. Write the app!
+- Fully functional CRUD on a Pokemon model (i.e., you can read, create, update and delete a Pokemon).
 
 ## Recommendations
-- Start small, get your data working first
-- be able to CRUD Pokemon in a REPL
-- build a seed file so that you have data you can test your UI against
-- then build the UI
 
-## Resources for bonus
+- Start small. Get your data working first.
+- Be able to CRUD a Pokemon in Pry before doing that through Sinatra.
+- Build a seed file so that you have data you can test your UI against.
+- Build the UI after the previous steps have been completed.
```

Comment:
@jshawl This is a lot of data. Do you think it might end up confusing them? Don't want them to think they have to create columns for these attributes.

https://raw.githubusercontent.com/ga-wdi-exercises/pokemon_underscore/master/js/src/data.js